kits/default/bcl-builder.config.js

```diff
@@ -17,5 +17,6 @@ module.exports = {
         sourceMap: "file",
       },
     },
-  ]
+  ],
+  copy: []
```

Comment:
Only remark that at alexis branch there are more changes not included here:
![image](https://user-images.githubusercontent.com/6751237/131801298-901cd4eb-a25a-43a6-9bf2-5404782715c8.png)

Reply:
that part of his branch is outdated

Reply:
I also have a warning on dropdown pattern, but the dropdown works. The rest of patterns seems to work ok.

![image](https://user-images.githubusercontent.com/6751237/131801971-5a276da0-1347-4f48-a147-18fa79efc230.png)